pkg/config/application_ecs.go

```diff
@@ -14,7 +14,11 @@
 
 package config
 
-import "encoding/json"
+import (
+	"encoding/json"
+
+	"github.com/aws/aws-sdk-go-v2/service/ecs/types"
```

Comment:
Should convert this types.AwsVpcConfiguration to platformprovider ecs types so that we can remove the depending on 3rd package from this configuration package.